dnf-behave-tests/common/cmd.py

```diff
@@ -60,6 +60,19 @@ def file_has_mode(context, filepath, octal_mode_str):
     assert oct(octal_mode) == oct(octal_file_mode), \
         "File \"{}\" has mode \"{}\"".format(matched_files[0], oct(octal_file_mode))
 
+@behave.step("file \"{filepath}\" has ACL entry \"{entry}\"")
```

Comment:
Rather than using getfacl, what about using pylibacl?  python-pylibacl package.  For example:

```
def file_has_acl_entry(context, filepath, entry):
    import posix1e
    filepath = prepend_installroot(context, filepath)
    acl = posix1e.acl(file=filepath)
    for line in acl.to_any_text().split("\n"):
        if line == entry:
            return
    raise AssertionError("File \"{}\" has ACL:\n{}".format(filepath, acl.to_any_text()))
```